Update sample Java client

ludwix opened the pull request:
https://jira.voicepin.com/browse/VPIN-772

Updated paths, enums and constants.

Commit:
Update sample Java client

https://jira.voicepin.com/browse/VPIN-772

Updated paths, enums and constants.

Commit:
Update sample Java client

https://jira.voicepin.com/browse/VPIN-772

Reverted paths.

kodrzywolek commented on src/main/java/com/voicepin/example/client/VoicepinClient.java:
```diff
@@ -29,8 +29,8 @@
-    private static final String EXTERNAL_ID_QUERY = "extId";
-    private static final String API_KEY_QUERY = "apiKey";
+    public static final String USER_IDENTIFIER = "speakerId";
+    private static final String API_KEY = "apiKey";
```
Spróbuj zmienić nazwę API_KEY. Obecna sugeruję że pod tą zmienną kryje się stałe apiKey, stara nazwa też nie była świetna jakby co. Może wydziel wszystkie te stałe do statycznej klasy i tą klasę nazwij tak aby było wiadomo co się w niej kryje. To tylko jedna z opcji.

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/AddResponse.java:
```diff
@@ -5,7 +5,7 @@
 public class AddResponse extends Response {
 
-    @JsonProperty("VoiceprintId")
+    @JsonProperty("voiceprintId")
```
Chyba nie ma sensu trzymać stringa w adnotacji skoro pole nazywa się tak samo.

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/EnrollResponse.java:
```diff
@@ -4,10 +4,10 @@
     protected boolean isTrained;
     
-    @JsonProperty("EnrollStatus")
+    @JsonProperty("enrollStatus")
```
Chyba nie ma sensu trzymać stringa w adnotacji skoro pole nazywa się tak samo.

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/EnrollResponse.java:
```diff
@@ -4,10 +4,10 @@
 public class EnrollResponse extends Response {
 
-    @JsonProperty("IsTrained")
+    @JsonProperty("trained")
```
Zmień też nazwę zmiennej na trained, pamiętaj o getterach i setterach, aby nie było metod typu isIsTrained()

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/VerifyResponse.java:
```diff
@@ -4,10 +4,10 @@
 public class VerifyResponse extends Response {
 
-    @JsonProperty("Score")
+    @JsonProperty("score")
```
Chyba nie ma sensu trzymać stringa w adnotacji skoro pole nazywa się tak samo. Niżej to samo.

Commit:
Update sample Java client

https://jira.voicepin.com/browse/VPIN-772

Corrections after pull requests (paths consts, json propety names )

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/EnrollResponse.java:
```diff
@@ -4,18 +4,18 @@
+        return trained;
     }
 
     public void setIsTrained(boolean value) {
```
setIsTrained -> setTrained

kodrzywolek commented on src/main/java/com/voicepin/example/client/messages/EnrollResponse.java:
```diff
@@ -4,18 +4,18 @@
 
-    @JsonProperty("IsTrained")
-    protected boolean isTrained;
+    @JsonProperty("trained")
```
Teraz możesz usunąć stringa z adnotacji

Commit:
Update sample Java client

https://jira.voicepin.com/browse/VPIN-772

Corrections after pull requests - response messages.
